.changeset/tough-melons-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': major
```

Comment:
```suggestion
'@shopify/polaris-icons': minor
```

This is an update to documentation so it should be a minor change

Reply:
ah yes, i could not figure out how to _not_ make it major! thank you!